### ISSUE TITLE
Just asking

### ISSUE THREAD

**Issue by ridom** [wjcrowcroft/MotionCAPTCHA#22]
Last update is 6 years old, it works fine, it recognize good or bad schema, my questions are :

What is wrong with this motion captcha ? Lack of accessibility ? Not mobile friendly ?
Why isn't there more engagement with it? Technical issue that allows bots to pass anyway ?

**Comment by s0kil** [comment]
Wish this awesome library had more support.
